### DISCUSSION TITLE
deploy cubefs  on k8s  error

### DISCUSSION THREAD

**Discussion by wantWhatBike** [cubefs/cubefs#2561] [General]
when i deploy cubefs on k8s with helm,  the master pod  crashed,  here is the error log:
kubectl logs -f -n cubefs master-0
Defaulted container "master-pod" out of: master-pod, init-config (init)
{
  "role": "master",
  "ip": "10.65.220.13",
  "listen": "17010",
  "prof": "17020",
  "id": "1",
  "peers": "1:master-0.master-service:17010",
  "retainLogs": "2000",
  "logDir": "/cfs/logs",
  "logLevel": "error",
  "walDir": "/cfs/data/master/raft",
  "storeDir": "/cfs/data/master/rocksdbstore",
  "consulAddr": "http://consul-service:8500",
  "exporterPort": "9500",
  "clusterName": "my-cluster",
  "metaNodeReservedMem": "67108864"
}
start master
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /cfs/bin/cfs-server)
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /cfs/bin/cfs-server)
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by /cfs/bin/cfs-server)
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /cfs/bin/cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by ./cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by ./cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by ./cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by ./cfs-server)

**Comment by joseluisll** [comment]
I experienced this error as well. It means the base centos 7 image has an old libstdc++ version. The only way forward i found is to create a base image using ubuntu:jammy, because i Know debian based distros better.
